apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V25__Add_partd_contract_number_index.sql

```diff
@@ -0,0 +1,36 @@
+
+create index ${logic.index-create-concurrently} "Beneficiaries_partd_contract_number_jan_id_idx"
+    on "Beneficiaries" ${logic.using-hash} ("partDContractNumberJanId");
+
+create index ${logic.index-create-concurrently} "Beneficiaries_partd_contract_number_feb_id_idx"
+    on "Beneficiaries" ${logic.using-hash} ("partDContractNumberFebId");
+
```

Comment:
Did you consider a partial index (ie. an index with a where clause)? Looking at the shape of these columns, the data look's very sparse, with `0`,`N`, and `C` being the most common values. Real contracts like `H0023` only make a small ~10% of the data. If possible, using a partial index will make the index much smaller and perform much better.